src/shared/codelens/pythonCodeLensProvider.ts

```diff
@@ -51,56 +51,16 @@ const getSamProjectDirPathForFile = async (filepath: string): Promise<string> =>
     return path.dirname(filepath)
 }
 
-async function getLambdaHandlerCandidates({
-    uri,
-    pythonSettings
-}: {
-    uri: vscode.Uri
-    pythonSettings: SettingsConfiguration
-}): Promise<LambdaHandlerCandidate[]> {
-    const PYTHON_JEDI_ENABLED_KEY = 'jediEnabled'
-    const RETRY_INTERVAL_MS = 1000
-    const MAX_RETRIES = 10
-
-    const logger = getLogger()
+async function getLambdaHandlerCandidates(uri: vscode.Uri): Promise<LambdaHandlerCandidate[]> {
     const filename = uri.fsPath
 
-    let symbols: vscode.DocumentSymbol[] =
+    const symbols: vscode.DocumentSymbol[] =
```

Comment:
this can be collapsed into the return as well now

Reply:
The fake null coalescing makes it awkward.